src/Sinch/Fax/Services/UpdateServiceRequest.cs

```diff
@@ -0,0 +1,113 @@
+using System.Text;
+using System.Text.Json.Serialization;
+using Sinch.Fax.Faxes;
+
+namespace Sinch.Fax.Services
+{
+    public sealed class UpdateServiceRequest
```

Comment:
This class contains the same fields as `CreateServiceRequest` + the `id`. Do you think it would be valuable to use inheritance?